install/kubernetes/helm/istio/values-envoyv2-transition.yaml

```diff
@@ -19,7 +19,7 @@ ingress:
 zipkin:
   enabled: true
 
-sidecar-injector:
+sidecarInjectorWebhook:
```

Comment:
can we have it enabled by default ? Only issue would be k8s pre 1.9, where it may fail to start - but I would optimize for 1.9+ ux.